.settings/com.google.gdt.eclipse.core.template.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+jarsExcludedFromWebInfLib=*/src/test/resources/lib/appengine/appengine-api-labs.jar|*/src/test/resources/lib/appengine/appengine-api-stubs.jar|*/src/test/resources/lib/appengine/appengine-api.jar|*/src/test/resources/lib/appengine/appengine-remote-api.jar|*/src/test/resources/lib/appengine/appengine-testing.jar|*/src/test/resources/lib/httpunit/httpunit.jar|*/src/test/resources/lib/javamail/mail.jar|*/src/test/resources/lib/selenium/selenium-server-standalone-2.46.0.jar
+warSrcDir=src/main/webapp
+warSrcDirIsOutput=true
```

Comment:
Instead of providing a template file, just ask the user to copy paste the missing bit into the file? Furthermore, this setting can be an optional setting for core developers. Contributors need not go through this extra step.

Another thing: is there no way to give relative paths here?